_posts/2020-11-05-markdown-tutorial.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Markdown Tutorial"
+date:   2020-11-06 12:54:04 -0700
+categories: markdown tutorial
+---
+In this post, I will be writing a tutorial on how to create a Markdown from scratch and all that a beginner needs to know about it.
+
+Markdown is a lightweight Markup language with easy to use plain text formatting syntax that was created by John Gruber and Aaron Swartz in 2004.
+It is used for styling different forms of writing like formatting README files, writing messages in online discussions platform, writing content for the web and to create rich text.
+
+## Basic Markdown Syntaxes
+Syntaxes are set of rules for grammar and spelling that defines the combination of symbols considered to be correctly structured in a particular programming language.
+
+##### Headings
+Headings in Markdown are any line which is prefixed with the hashtag (#) symbol. The number of hashtags represents the heading level. A total of 6 levels can be made with 6 hashtags but in most writings, only 3 is commonly used. The syntax in block quotes are showing what the text looks like in Markdown.
+
+```
+# Heading 1
+## Heading 2  
+### Heading 3
```

Comment:
I would try to mention the advantages of why you would want to use different headers or put a photo showing that the headers are a good way to subdivide various topics you wish to discuss.

Reply:
@Alicebada I agree, I think its a good idea to say headings are useful to breakup and organize the report!